scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -42,10 +49,11 @@ def cpuflags():
 
 
 def hypervisor_has_virtio(libvirt_type):
-    if (libvirt_type == "xen" or libvirt_type == "hyperv"):
-        return False
-    else:
+
+    if libvirt_type == "kvm":
```

Comment:
Actually @aspiers is right. Your patch transform the method into this:

```Python
def hypervisor_has_virtio(libvirt_type):

    if libvirt_type == "kvm":
        return True
    else:
        return False
```

That is equivalent to this:

```Python
def hypervisor_has_virtio(libvirt_type):
    return libvirt_type == 'kvm'
```

A different issue is if 'kvm' is the only one `libvirt_type` that allows virtio : ?